test/link/import_memory_mismatch.txt

```diff
@@ -0,0 +1,41 @@
+;;; TOOL: run-wasm-link
```

Comment:
This test seems to be the same as import_memory.txt above, or am I missing something?